config/default.cfg

```diff
@@ -344,8 +344,8 @@ cfg$gms$cm_gs_ew              <- 20      # def <- 20
 cfg$gms$cm_LimRock             <- 1000    # def <- 1000
 cfg$gms$c_tau_so2_xmpt       <- 0       # def <- 0
 cfg$gms$cm_expoLinear_yearStart <- 2050  # def <- 2050
-cfg$gms$c_budgetCO2FFI       <- 1000     # def <- 1000
-cfg$gms$c_budgetCO2          <- 1350     # def <- 1350
+cfg$gms$c_budgetCO2FFI       <- 700      # def <- 700
```

Comment:
I computed the emissions 2011-2020 (excluding 2020) that were added to the previous formula, and it was around 300 Gt.